suite-common/wallet-core/package.json

```diff
@@ -25,7 +25,8 @@
         "@suite-common/wallet-utils": "workspace:*",
         "@trezor/connect": "workspace:9.0.5",
         "@trezor/utils": "workspace:*",
-        "date-fns": "^2.29.3"
+        "date-fns": "^2.29.3",
+        "proxy-memoize": "^2.0.0"
```

Comment:
it should be listed in `packages/suite/package.json` file too, right?

Reply:
It should but it's even stranger that eslint doesn't complain about that.

Reply:
Fixed in https://github.com/trezor/trezor-suite/pull/7413/commits/f2f2ac17785626cde3fbc2bfbff787ddd1f244b2